doc/cascadia/profiles.schema.json

```diff
@@ -817,6 +819,63 @@
         }
       ]
     },
+    "GlobalSummonAction": {
+      "description": "Arguments corresponding to a focusPane Action",
+      "allOf": [
+        { "$ref": "#/definitions/ShortcutAction" },
+        {
+          "properties": {
+            "action": { "type": "string", "pattern": "globalSummon" },
+            "desktop":{
+              "type":"string",
+              "default":"toCurrent",
+              "description":"This controls how the terminal should interact with virtual desktops.\n- \"any\": Leave the window on whichever desktop it's already on - will switch to that desktop as the window is activated.\n- \"toCurrent\" (default): Move the window to the current virtual desktop.\n- \"onCurrent\": Only summon the window if it's already on the current virtual desktop. ",
+              "enum":[
+                "any",
+                "toCurrent",
+                "onCurrent"
+              ]
+            },
+            "monitor":{
+              "type":"string",
+              "default":"toMouse",
+              "description":"This controls the monitor that the window will be summoned from/to.\n- \"any\": Summon the most recently used window, regardless of which monitor it's currently on.\n- \"toCurrent\": Summon the most recently used window to the monitor with the current foreground window.\n- \"toMouse\" (default): Summon the most recently used window to the monitor where the mouse cursor is.",
+              "enum":[
+                "any",
+                "toCurrent",
+                "toMouse"
+              ]
+            },
+            "name":{
+              "type":"string",
+              "description":"When provided, summon the window whose name or ID matches the given name value. If no such window exists, then create a new window with that name."
+            },
+            "dropdownDuration":{
+              "type":"number",
+              "minimum": 0,
+              "default":0,
+              "description":"When provided with a positive number, \"slide\" the window in from the top of the screen using an animation that lasts dropdownDuration milliseconds."
+            },
+            "toggleVisibility":{
+              "type":"boolean",
+              "default":true,
+              "description":"When true, pressing the assigned keys for this action will dismiss (minimize) the window when the window is currently the foreground window."
+            }
+          }
+        }
+      ]
+    },
+    "QuakeModeAction": {
+      "description": "This action is a special variation of the globalSummon action. It specifically summons the quake window,If you'd like to change the behavior of the quakeMode action, we recommended creating a new globalSummon entry",
```

Comment:
Should this say `the _quake window`? [The documentation](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#open-the-quake-mode-window-preview) doesn't, but it instead has a hyperlink to an article that stipulates the `_quake` name.

The punctuation before and after the last sentence is not right.